atlasdb-coordination-impl/src/main/java/com/palantir/atlasdb/coordination/CoordinationStore.java

```diff
@@ -60,5 +60,5 @@ default boolean isInitialized() {
      *
      * @throws IllegalStateException if the proposal fails, but no current value exists
      */
-    CheckAndSetResult<ValueAndBound<T>> transformAgreedValue(Function<ValueAndBound<T>, T> transform);
+    TransformResult<ValueAndBound<T>> transformAgreedValue(Function<ValueAndBound<T>, T> transform);
```

Comment:
Let's fix the Javadoc to reflect the changes / breaking of CAS dependency

Reply:
I almost want to remove this sentence, but lemme try to rephrase.

Reply:
No, I will remove both. What we're trying to say is that the returned value is basically the current value and that's now fairly clear from the actual TransformResult.